### DISCUSSION TITLE
Problem with chat

### DISCUSSION THREAD

**Discussion by MichalSoprano** [runelite/runelite#17262] [Q&A] [answered]
I must click "enter + shift" when I'd like send a message or write to chat instead of just enter

**Comment by YvesW** [comment, marked as answer]
Github is not a support forum, join the discord for support to have this problem resolved.
![221373292-659c7b87-eef2-400b-95e3-e1974c6535a9](https://private-user-images.githubusercontent.com/41973452/289179311-ad9ff9f5-6d9b-4490-8276-cfe3f8bca29e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIwNjk0MzYsIm5iZiI6MTcwMjA2OTEzNiwicGF0aCI6Ii80MTk3MzQ1Mi8yODkxNzkzMTEtYWQ5ZmY5ZjUtNmQ5Yi00NDkwLTgyNzYtY2ZlM2Y4YmNhMjllLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA4VDIwNTg1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI3OTc3MjcwZGVhMzc4NTNiNjMyMzQzM2ZiZGVkYzdmMmRlMjk2NGNiYWU4MDM5ZWY2ZjBjYWFhY2U5NDE2OTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rdxNla7J3Kf3HjSDPZ8tzOLOjixDdWXEoXmFMwQadCA)
